sandy/gls.py

```diff
@@ -278,3 +281,47 @@ def ishikawa_factor(S, Vx_prior, Vy_extra):
     Vy_extra_ = np.diag(pd.DataFrame(Vy_extra).values)
     index = pd.DataFrame(Vy_extra).index
     return pd.Series(Vy_values / Vy_extra_, index=index)
+
+
+def constrained_ls_update(x_prior, S, Vx_prior, threshold=None):
```

Comment:
don't we miss the extra info? maybe not...if you write the equation it is easier to understand what you are trying to implement

Reply:
In the constrained Least Square method we dont have extra information. For covariance matrix is the same formula that for GLS, but without V_extra

Reply:
ok, just write the equation in the docstring, then I know what you are doing